easybuild/easyblocks/h/hdf5.py

```diff
@@ -51,12 +52,16 @@ def configure_step(self):
             ("Szip", "--with-szlib"),
             ("zlib", "--with-zlib"),
         ]
+        filter_deps = build_option('filter_deps')
         for (dep, opt) in deps:
             root = get_software_root(dep)
             if root:
                 self.cfg.update('configopts', '%s=%s' % (opt, root))
             else:
-                raise EasyBuildError("Dependency module %s not loaded.", dep)
+                if dep in filter_deps:
+                    self.cfg.update('configopts', '%s' % opt)
+                else:
+                    raise EasyBuildError("Dependency module %s not loaded.", dep)
```

Comment:
I guess it's like this since this was written long before `--filter-deps`...

Let's go with the non-error option instead then?